docs/azure_arc_jumpstart/azure_arc_vsphere/resource_bridge/powershell/_index.md

```diff
@@ -0,0 +1,197 @@
+  - _'gateway'_ - Gateway address to be used for the appliance
+  - _'ipAddressPrefix'_ - Network address in CIDR notation to be used by the appliance
+  - _'k8sNodeIpPoolStart'_ - IP range start for the IPs to be used by the appliance
+  - _'k8sNodeIpPoolEnd'_ - IP range end for the IPs to be used by the appliance
+  - _'segment'_ - Name of the virtual network or segment to which the appliance VM must be connected
+  - _'resourcePool'_ - the name of the resource pool to be used by the appliance
+  - _'controlPlaneEndpoint'_ - IP address of the Kubernetes cluster control plane
+  - _'vSphereRP'_ - ConnectedVMwarevSphere resource provider Id 
+  
+  ![Screenshot environment variables](./02.png)
+
+- Once you have provided all of the required environment variables. Open an administrative PowerShell window and run the script with the command:
+
+  ```powershell
+  .\vCenter_onboarding.ps1
+  ```
+
+  ![Screenshot showing script output](./03.png)
+
+  ![Screenshot showing script output](./04.png)
+
+  ![Screenshot showing script output](./05.png)
+
+  ![Screenshot showing script output](./06.png)
+
+- While the script is running, from vCenter you should be able to see a running task:
+
+  ![Screenshot showing vCenter task](./07.png)
+  
+  ![Screenshot showing vCenter task](./08.png)
+  
+  ![Screenshot showing script output](./10.png)
+  
+   ![Screenshot showing script output](./11.png)
+   
+   ![Screenshot showing script output](./12.png)
+    
+   ![Screenshot showing script output](./13.png)
+     
+   ![Screenshot showing script output](./14.png)
+   
+   ![Screenshot showing script output](./15.png)
+   
+   ![Screenshot showing script output](./16.png)
+   
+   ![Screenshot showing script output](./17.png)
+
+- From the Azure Portal, in the resource group you should see three new resources, including a VMware vCenter:
+
+  ![Resources in Portal](./18.png)
+
+  ![Connected VMware vCenter](./19.png)
+
+- You should also be able to get a list of VMs, resource pools, templates, networks and data stores that are managed by the vCenter:
+
+  ![VM list](./20.png)
+  
+  ![Resource pools list](./21.png)
+  
+  ![Templates list](./22.png)
+  
+  ![Networks list](./23.png)
+  
+  ![Data stores list](./24.png)
+
+## Clean up environment
+
+Complete the following steps to clean up your environment:
+
+- If you want to delete the entire Azure resources, simply delete the deployment resource group from the Azure portal.
+     
+    ![Screenshot showing Azure resource group deletion](./25.png)
+    
+- From the vSphere client, power Off the appliance and remove from disk.
+
+    ![Screenshot showing powering off the appliance from vSphere client](./26.png)
+
+    ![Screenshot showing deleting the appliance from vSphere client](./27.png)
+    
+- From the vSphere client, remove the appliance template from disk.
+
+    ![Screenshot showing deleting the appliance template from vSphere client](./28.png)
```

Comment:
The step numbers in these screenshots are hard to see. Consider using something like that:

![image](https://user-images.githubusercontent.com/46730876/166928118-1d7e68e5-92de-4897-89f0-2d38022453dd.png)